### ISSUE TITLE
Continuous integration broken on Python 2.7 / PySide + Qt

### ISSUE THREAD

**Issue by mdickinson** [enthought/enable#275]
Several tests currently fail with errors resembling the following:
```
======================================================================
ERROR: Failure: ImportError (Unable to import the image backend for the qt4 toolkit (reason: ['ImportError: No module named PySide.QtOpenGL\n']).)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python2.7_with_system_site_packages/local/lib/python2.7/site-packages/nose/loader.py", line 414, in loadTestsFromName
    addr.filename, addr.module)
  File "/home/travis/virtualenv/python2.7_with_system_site_packages/local/lib/python2.7/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/home/travis/virtualenv/python2.7_with_system_site_packages/local/lib/python2.7/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/travis/build/enthought/enable/enable/savage/svg/tests/test_document.py", line 6, in <module>
    from enable.savage.svg.backends.kiva.renderer import Renderer as KivaRenderer
  File "/home/travis/build/enthought/enable/enable/savage/svg/backends/kiva/renderer.py", line 7, in <module>
    from enable.compiled_path import CompiledPath as KivaCompiledPath
  File "/home/travis/build/enthought/enable/enable/compiled_path.py", line 17, in <module>
    from .toolkit import toolkit_object
  File "/home/travis/build/enthought/enable/enable/toolkit.py", line 47, in <module>
    _init_toolkit()
  File "/home/travis/build/enthought/enable/enable/toolkit.py", line 40, in _init_toolkit
    format_exception_only(t, v))
ImportError: Unable to import the image backend for the qt4 toolkit (reason: ['ImportError: No module named PySide.QtOpenGL\n']).
```
(See the CI runs for #272 and #273 for a complete list of the failing tests.)

[See also #274]

**Comment by corranwebster** [comment]
I think this particular issue was fixed by https://github.com/enthought/pyface/pull/271.  There is a secondary issue with fonttools.

**Comment by mdickinson** [comment]
@corranwebster: Yep, looks like it. Thanks. And it looks as though the fonttools issue was fixed in #276.

CI seems to be passing on 2.7 now. Closing this issue.
